web/src/components/Plugin/data.tsx

```diff
@@ -26,3 +26,8 @@ export const PLUGIN_ICON_LIST: Record<string, any> = {
   'openid-connect': <IconFont name="iconicons8-openid" />,
   'kafka-logger': <IconFont name="iconApache_kafka" />,
 };
+
+export const PLUGIN_FILTER_LIST: Record<string, { list: PluginComponent.ReferPage[] }> =
```

Comment:
since we have filterd the `redirect` plugin, do we have plan to filter the `proxy-rewite` or some other plugins which are frequently used in the future? @LiteSun @juzhiyuan

Reply:
Currently, the redirect plugin is extracted from the route page step3 into step1, so it needs to be filtered out in step3.

It may be necessary to create an issue to discuss whether other common plugins need to be filtered out.

Reply:
Could we use a global `config` file to store those rules? or coming contributors may don't know this tip.

Reply:
IMHO, add a comment tip is fine.